packages/sync/src/useMultiplayerSync.ts

```diff
@@ -155,13 +167,32 @@ export function useMultiplayerSync(
 	)
 }
 
-/** @public */
+/**
+ * Options for the {@link useMultiplayerSync} hook.
+ * @public
+ */
 export interface UseMultiplayerSyncOptions {
+	/**
+	 * The URI of the multiplayer server. This must include the protocol, e.g. `wss://server.example.com` or `http://localhost:5858`.
+	 */
 	uri: string
-	roomId?: string
+	/**
+	 * The user information. If not provided, a default implementation based on localStorage is used.
+	 * This should be synchronized with the configuration for the main `<Tldraw />` component.
+	 */
 	userPreferences?: Signal<TLUserPreferences>
```

Comment:
mmmmm we should add an example around this

Reply:
yes. i also want to make it optional to pass in a non-signal here. will give that a little go and see if it can be done well quickly.